### PR TITLE
add OSTH

### PR DESCRIPTION
**Pull request by jason3e7**
add Offensive Security Threat Hunting Certification

### CODE REVIEW

File: src/OSTH-exam-report-template_OS_v1.md [hunk 1]
```diff
@@ -0,0 +1,178 @@
...
+
+1. PC1 was used as the initial entry vector by the threat actor by trying numerous passwords against several user accounts. The threat actor finally succeeded and got access to this machine with administrative privileges. 
+2. PC2 was configured to use the same password for the local administrator account and the threat actor used it to get access to it. On the machine, the attacker obtained credentials from logged on users by using Mimikatz.
+3. PC3 was accessed using one of the obtained sets of credentials from PC2. The threat actor accessed and exfiltrated the secret chocolate muffin recipe from this machine. 
+
+## Recommendations
+
+The threat hunt revealed the following high-level path the threat actor took to compromise the Megacorp One environment: 
+
+- Escalate Incident to Incident Response Team:
```
Comment (noraj) [inline src/OSTH-exam-report-template_OS_v1.md:75]:
```suggestion
1. **Escalate Incident to Incident Response Team:**
```

File: src/OSTH-exam-report-template_OS_v1.md [hunk 2]
```diff
@@ -0,0 +1,178 @@
...
+  - Escalate the incident to the incident response team to conduct a thorough investigation of the identified compromises. The focus should be on assessing the full scope of the incident and understanding its impact on the organization's systems and data.
+  - Collaborate closely with the incident response team to share findings, provide context, and support their efforts in containing and remediating the security incidents.
+- Continued Support and Analysis:
```
Comment (noraj) [inline src/OSTH-exam-report-template_OS_v1.md:78]:
```suggestion
2. **Continued Support and Analysis:**
```

File: src/OSTH-exam-report-template_OS_v1.md [hunk 3]
```diff
@@ -0,0 +1,178 @@
...
+  - Remain actively involved in supporting other defensive teams, particularly during the incident detection and identification phase of the incident management process.
+  - Conduct further analysis of the malware samples to extract additional IoCs and behavioral patterns, providing valuable insights for ongoing threat detection and mitigation efforts.
+- Continuous Improvement and Training:
```
Comment (noraj) [inline src/OSTH-exam-report-template_OS_v1.md:81]:
```suggestion
3. **Continuous Improvement and Training:**
```

File: src/OSTH-exam-report-template_OS_v1.md [hunk 4]
```diff
@@ -0,0 +1,178 @@
...
+01/09/2024 3:58:00 PM | End of Password Spraying. | Host: PC1
+01/09/2024 3:58:15 PM | Successful login for local Administrator user | Host: PC1 User: Administrator (local)
+01/09/2024 3:59:00 PM | Download of meterpreter.exe from `<IP>` via Browser | Host: PC1 User: Administrator
+01/09/2024 3:59:49 PM | Process Creation of meterpreter.exe | Host: PC1 User: Administrator (local)
+01/09/2024 4:05:11 PM | Process Creation of PsExec | Host: PC1 User: Administrator (local) Target Machine: PC2 Target User: Administrator (local) Password: Password1!"
+... | ... | ...
+01/11/2024 1:11:11 AM | Process Creation of Zwetsch.exe | Host: PC2 User: Administrator (local)
+... | ... | ...
+
+
```
Comment (noraj) [inline src/OSTH-exam-report-template_OS_v1.md:146]:
```suggestion
```

File: src/OSTH-exam-report-template_OS_v1.md [hunk 5]
```diff
@@ -0,0 +1,178 @@
...
+Our hunting hypothesis was:
+
+We suspect that PC3 and PC2 are not the only systems compromised by the WAG threat actor. While we couldn’t identify any further indicators that revealed additional compromised systems using the credentials obtained from PC2, or following the compromise of PC3, it is likely that PC2 was not the initial system compromised by WAG, given that it is not externally accessible. Therefore, we suspect that at least one other machine is compromised. We will validate this by investigating the events preceding the use of Mimikatz to obtain credentials and by identifying the vector the threat actor used to access PC2 and breach the perimeter.
+
+# Hunt Narrative
+
+The threat intelligence report covering TTPs of the threat actor We Are Garfield provided a list of IoCs including SHA-256 hashes. We used the following query in Splunk to hunt for these hashes:
+
+```
```
Comment (noraj) [inline src/OSTH-exam-report-template_OS_v1.md:107]:
```suggestion
```default
```

https://github.com/noraj/OSCP-Exam-Report-Template-Markdown/blob/master/FAQ.md#syntax-highlight-style-ignored-when-no-language-provided

File: src/OSTH-exam-report-template_OS_v1.md [hunk 6]
```diff
@@ -0,0 +1,178 @@
...
+
+The event provides us several important information that can be leveraged in our hunt:
+
+- Username: Administrator
+- Filename: Zwetsch.exe
+- Directory: `C:\hackingtools\`
+
+Based on the matching SHA-256 hash of the threat intelligence report and the characteristic commandline argument “sekurlsa::logonpasswords”, we can be certain that this is Mimikatz. 
+
+...
```
Comment (noraj) [inline src/OSTH-exam-report-template_OS_v1.md:130]:
```suggestion
[…]
```

File: src/OSTH-exam-report-template_OS_v1.md [hunk 7]
```diff
@@ -0,0 +1,178 @@
...
+
+ Timestamp            | Observation | Affected Assets
+----------------------|-------------|-----------------
+01/09/2024 3:25:00 PM | Beginning of Password Spraying with Password Password1! | Host: PC1
+01/09/2024 3:58:00 PM | End of Password Spraying. | Host: PC1
+01/09/2024 3:58:15 PM | Successful login for local Administrator user | Host: PC1 User: Administrator (local)
+01/09/2024 3:59:00 PM | Download of meterpreter.exe from `<IP>` via Browser | Host: PC1 User: Administrator
+01/09/2024 3:59:49 PM | Process Creation of meterpreter.exe | Host: PC1 User: Administrator (local)
+01/09/2024 4:05:11 PM | Process Creation of PsExec | Host: PC1 User: Administrator (local) Target Machine: PC2 Target User: Administrator (local) Password: Password1!"
+... | ... | ...
```
Comment (noraj) [inline src/OSTH-exam-report-template_OS_v1.md:142]:
```suggestion
[…] | […] | […]
```

File: src/OSTH-exam-report-template_OS_v1.md [hunk 8]
```diff
@@ -0,0 +1,178 @@
...
+01/11/2024 1:11:11 AM | Process Creation of Zwetsch.exe | Host: PC2 User: Administrator (local)
+... | ... | ...
```
Comment (noraj) [inline src/OSTH-exam-report-template_OS_v1.md:144]:
```suggestion
[…] | […] | […]
```

File: src/OSTH-exam-report-template_OS_v1.md [hunk 9]
```diff
@@ -0,0 +1,178 @@
...
+
+Attached is a compiled list of the resulting  IOCs found during the threat hunting sprint.
+
+**File Hashes**
+
+File Name       | SHA256
+----------------|------------------------------------------------------------------
+Zwetsch.exe     | 4ED877F6F154EB6EBB02EE44E4D836C28193D9254A4A3D6AF6236D8F5BAB88D2
+meterpreter.exe | DF99BBABE7BD0E7A1D96CF370B78FDCF250AF380065A3D51F57EDE6A571E2C15
+...             | ...
```
Comment (noraj) [inline src/OSTH-exam-report-template_OS_v1.md:167]:
```suggestion
[…]             | […]
```

File: src/OSTH-exam-report-template_OS_v1.md [hunk 10]
```diff
@@ -0,0 +1,178 @@
...
+
+**Network Communications**
+
+Type           | Value
+---------------|----------------------------------------
+C&C            | 192.168.1.1:9999 (meterpreter.exe)
+Exfiltration   | 192.168.1.1:80 (WebDAV Share “looty”)
+File Download  | 192.168.1.1:80 (meterpreter.exe)
+File Download  | 192.168.1.1:80 (Zwetsch.exe)
+...            | ...
```
Comment (noraj) [inline src/OSTH-exam-report-template_OS_v1.md:177]:
```suggestion
[…]            | […]
```

**Review by noraj — CHANGES_REQUESTED** [review]
Thanks a lot for your contribution! That's awesome.
I was very busy, sorry for the delay before a proper review.

**Comment by jason3e7** [comment]
Hi @noraj, I committed the changes as requested.
But the indentation changed from two spaces to four spaces.
Then ordered list and unordered sublist will be displayed correctly in the generated pdf.
OSTH-exam-report-template_OS_v1.md, line 75 ~ 85

**Pull request merged**